plugin/storage/grpc/options.go

```diff
@@ -33,6 +33,7 @@ const (
 	remotePrefix             = "grpc-storage"
 	remoteServer             = remotePrefix + ".server"
 	remoteConnectionTimeout  = remotePrefix + ".connection-timeout"
+	grpcMaxRecvMsgSize       = remotePrefix + ".max-recv-msg-size"
```

Comment:
```suggestion
	grpcMaxRecvMsgSize       = remotePrefix + ".max-message-size"
```
to be consistent with `cmd/collector/app/flags/flags.go:45`

Reply:
Would this be ambiguous that it also affects the sent message size? (although I don't see any chance a storage plugin client would send a large request anyway).

Reply:
why not apply the same setting both ways?

Reply:
Well basically I just didn't see the motivation for configuring the send size.

Reply:
the motivation is using the same param and not introducing a new name that does not match existing flags